reviews-block.php

```diff
@@ -0,0 +1,60 @@
+<?php
```

Comment:
Rename this file to `wp-plugin-review-block`

Reply:
Fixed.